custom_components/plugwise/climate.py

```
@@ -217,7 +212,11 @@ def hvac_mode(self) -> HVACMode:
 
     @property
     def hvac_modes(self) -> list[HVACMode]:
-        """Return a list of available HVACModes."""
+        """Return a list of available HVACModes.
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)